config/environments/production.rb

```diff
@@ -120,7 +120,7 @@
   # In production, we only accept CORS request from sumofus.org or its subdomains.
   config.middleware.insert_before 0, Rack::Cors, logger: (-> { Rails.logger }) do
     allow do
-      origins(%r{^(https?:\/\/)?([a-z0-9-]+\.)?sumofus\.org$}i)
+      origins(%r{^(https?:\/\/)?([a-z0-9-]+\.)*sumofus\.org$}i)
```

Comment:
this would catch any domain that ends in `sumofus.org` (including, for example, `fakesumofus.org`). For multiple domains you need to allow the capture group to repeat (with a `+`) so _I think_ this would work:

```diff
- ^(https?:\/\/)?([a-z0-9-]+\.)?sumofus\.org$
+ ^(https?:\/\/)?(([a-z0-9-]+\.)?)+sumofus\.org$
```
Basically, `(CAPTURE_GROUP_REGEXP)+` will allow the capture group (subdomain) to repeat infinitely. If you want to limit the number of times it repeats, you can use `{min,max}`. So if you want to limit it to at most 3 levels of subdomains, `(CAPTURE_GROUP_REGEXP){1,3}`. 


```regex
^(https?:\/\/)?(([a-z0-9-]+\.)?){1,3}sumofus\.org$
```
☝️  that will match:

```
sumofus.org
x.sumofus.org
x.y.sumofus.org
x.y.z.sumofus.org
```

but it woun't match `w.x.y.z.sumofus.org`.

Reply:
Ah of course. Thanks, Vince.